api/cpp/include/slint-platform.h

```diff
@@ -148,6 +148,14 @@ class Platform
     /// Returns a new WindowAdapter
     virtual std::unique_ptr<WindowAdapter> create_window_adapter() = 0;
 
+    // see internal/core/platform.rs
+    enum class Clipboard : uint8_t {
```

Comment:
This enum could also be automatically generated from the Rust enum. Let me know if you're interested in digging into that, then I can guide you, otherwise I can do that in a follow-up change.

Reply:
figured it out :+1: 
I'd actually done it right before... I just didn't using the cbindgen_private namespace.
thanks for the offer, though :)